model/src/main/kotlin/PackageCurationData.kt

```diff
@@ -127,7 +132,8 @@ private fun applyCurationToPackage(targetPackage: CuratedPackage, curation: Pack
         binaryArtifact = curation.binaryArtifact ?: base.binaryArtifact,
         sourceArtifact = curation.sourceArtifact ?: base.sourceArtifact,
         vcs = vcs,
-        isMetaDataOnly = curation.isMetaDataOnly ?: base.isMetaDataOnly
+        isMetaDataOnly = curation.isMetaDataOnly ?: base.isMetaDataOnly,
+        isModified = curation.isModified ?: base.isMetaDataOnly
```

Comment:
`isModified = curation.isModified ?: base.isModified` instead of
`isModified = curation.isModified ?: base.isMetaDataOnly`.

Reply:
Oops, copy & past classic. Thanks @fviernau. @KorSin, can you please file a follow-up PR?

Reply:
I've filed a PR already.

Reply:
@fviernau already did it: https://github.com/oss-review-toolkit/ort/pull/3110